tools/svg-to-sprite

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env bash
+
+# Rudimentary safety check.
+[[ ! -f "tools/svg-to-sprite" ]] && echo 'Please run this from package root!' && return 1;
+
+# Define some paths.
+src_svg_dir=build/svg-clean
+src_sprite_dir=src/svg-sprite
+src_css_dir=src/css
+dest_sprite_dir=build/svg-sprite
+dest_sprite_filename="$dest_sprite_dir"/social-logos.svg
+
+# Make dir if it doesn't exist.
+mkdir -p "$dest_sprite_dir"
+
+# Generate SVG sprite file.
+npx svgstore -o "$dest_sprite_filename" --inline "$src_svg_dir"/*.svg
+
+# Make SVG more conformant.
+sed -E -i.bak 's|<svg><defs/>|<svg xmlns="http://www.w3.org/2000/svg">|' "$dest_sprite_filename" && rm "$dest_sprite_filename.bak"
```

Comment:
I wonder if we should also strip the `<rect x="0" fill="none" width="24" height="24"/>` back out of each svg. The grunt-based version didn't add that in until after it generated the `social-logos.svg` file.

Reply:
I came here to suggest the same thing. In general I would remove it from all the generated SVG files.

Reply:
After a fair amount of digging/research and confirming with design (p1710961977565099-slack-C0D96691V), it seems this modification is no longer relevant and can be chopped altogether. 🥳

Reply:
Removed in c3e1df1.